src/ParametersHandler/YarpImplementation/include/BipedalLocomotion/ParametersHandler/YarpImplementation.h

```diff
@@ -35,6 +35,7 @@ class YarpImplementation : public IParametersHandler
     std::unordered_map<std::string, std::shared_ptr<YarpImplementation>> m_lists; /**< Map containing
                                                                                      pointers to the
                                                                                      (asked) groups */
+    std::unordered_set<std::string> m_paramNames; /** Set containing all stored parameter names */
```

Comment:
To be honest I am not a big fan of separated data structures for dependent data. In this case, you need to make sure that this set is also updated in case a new parameter is set.

Reply:
After 4865a32 a separated data structure is no longer necessary